docs/serving/samples/cloudevents/cloudevents-rust/README.md

```diff
@@ -0,0 +1,82 @@
+# Vert.x + Rust + Actix-web
+
+A simple web app written in Rust using [Actix web](https://github.com/actix/actix-web) that can receive CloudEvents.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/serving/samples/cloudevents/cloudevents-rust
+```
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- This guide uses Rust musl toolchain to build the image in order to create
+  really small docker images. To install the Rust toolchain: [`rustup`](https://rustup.rs/).
+  To install musl support: [MUSL support for fully static binaries](https://doc.rust-lang.org/edition-guide/rust-2018/platform-and-target-support/musl-support-for-fully-static-binaries.html).
+
+## Build and deploy the sample
+
+To build the binary, run:
+
+```shell
+cargo build --target x86_64-unknown-linux-musl --release
```

Comment:
Can we do this in a multi-stage Dockerfile as the others?

Reply:
I would prefer to avoid it here, it's quite complex and rust users don't always prefer this approach since they basically lose all the local compilation caching

Reply:
I thought clearing the local compilation caching was sort of the point -- any local compiler hijinks would be ironed out and the result would be reproducible by building in Docker.

I'd also expect that Rustaceans who were familiar with both Rust and containers would probably write their own Dockerfile based on the reference, anyway.

Reply:
Compilation caches in rust are reproducible because they're per exact crate versions, so technically is reproducible. And you don't want to clean the compilation cache in rust, 10 mins every time for building an image is not nice :smile: 

> I'd also expect that Rustaceans who were familiar with both Rust and containers would probably write their own Dockerfile based on the reference, anyway. 

I think so, they just need to create a multi stage build starting from debian rust image